util/misc.go

```diff
@@ -430,7 +434,15 @@ type SequenceTable interface {
 // LoadTLSCertificates loads CA/KEY/CERT for special paths.
 func LoadTLSCertificates(ca, key, cert string) (tlsConfig *tls.Config, err error) {
 	if len(cert) == 0 || len(key) == 0 {
```

Comment:
Should it be if either is not set? I.e. should we not try to use the key if set? And maybe warn/fail if cert is given without a key?

Reply:
Currently if you set this to an invalid location the server will start and log an error. I think a fatal error would be good here. The same could be done for an invalid config. However this line is not something I added, so maybe this needs to be done in another PR?

Reply:
Please create a new issue for that, should be a small and simple fix I assume.

Reply:
https://github.com/pingcap/tidb/issues/26065